cmd/commands/account/command.go

```diff
@@ -308,7 +308,7 @@ func (c *command) parseToken(ctx *cli.Context) *string {
 }
 
 func (c *command) registerIdentity(identity string, token *string) {
-	err := c.tequilapi.RegisterIdentity(identity, token)
+	err := c.tequilapi.RegisterIdentity(identity, "", token)
```

Comment:
just curiosity: what is this command for? is it used somewhere?

Reply:
> just curiosity: what is this command for? is it used somewhere?

I am curios too about this, anyone could give an answer please?